go-script/main.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"fmt"
+	prisma "go-gin/prisma-client"
+)
+
+func main() {
+
+	client := prisma.New(&prisma.PrismaOptions{
+		Endpoint: "http://localhost:4466/go-script/dev",
+	})
+
+	title := "Draft Post"
+	content := "This is a draft post."
+	email := "alice@prisma.io"
+
+	post, err := client.CreatePost(&prisma.PostCreateInput{
+		Title:   &title,
+		Content: &content,
+		Author: &prisma.UserCreateOneWithoutPostsInput{
+			Connect: &prisma.UserWhereUniqueInput{
+				Email: &email,
+			},
+		},
+	},
+	).Exec()
+
+	if err != nil {
+		panic(err)
+	}
+
+	id := post.ID
+
+	fmt.Println("Created post with id", &id)
+	fmt.Println(" ")
+
+	published := false
+
+	posts, err := client.Posts(&prisma.PostsParams{
+		Where: &prisma.PostWhereInput{
+			IsPublished: &published,
+		},
+	}).Exec()
+
+	if err != nil {
+		panic(err)
```

Comment:
This more informativity

```suggestion
              log.Fatal(err)
		panic(err)
```